gradle.properties

```diff
@@ -3,7 +3,7 @@ kotlin.code.style=official
 # config
 
 # codegen
-smithyVersion=1.23.0
+smithyVersion=[1.23.0,1.24.0[
```

Comment:
I would rather prefer exact version than range. 

Any specific reason for range?

Reply:
Yes, that way we can support any patch-level within the same major-minor, e.g., `1.23.1`. Having said that we are in the process to release `1.24.0` so we might need a follow up change for that.